internal/commands/convert.go

```diff
@@ -0,0 +1,115 @@
+package commands
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	"github.com/ghodss/yaml"
+	"github.com/plexsystems/konstraint/internal/rego"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newConvertCommand() *cobra.Command {
```

Comment:
Please add a `Long` description that includes a notice that any Rego files already containing OPA Metadata package annotations will be skipped.

Reply:
i've added one, but I'm not happy with wording – any suggestions?

Reply:
I would use:

Converts legacy annotations to OPA Metadata Annotations. Policies that already have _any_ package-level OPA Metadata Annotations set will be skipped, even if the policy contains legacy annotations.